app/cmd/restore_to_file.go

```diff
@@ -240,7 +240,7 @@ func MergeSnapshotsToBackingFile(snapFilepath, backingFilepath string) error {
 
 func rebaseSnapshot(snapFilepath, backingFilepath string) error {
 	logrus.Infof("Start rebaseSnapshot %s -> %s", snapFilepath, backingFilepath)
-	_, err := iutil.ExecuteWithoutTimeout(QEMUImageBinary, []string{"rebase", "-u", "-b", backingFilepath, snapFilepath})
+	_, err := iutil.ExecuteWithoutTimeout(QEMUImageBinary, []string{"rebase", "-u", "-b", backingFilepath, "-F", DefaultOutputFormat, snapFilepath})
```

Comment:
Is there any possible compatibility issue?

Reply:
I think it's ok because this flag exists (after -b) for a long time, it just could be omitted before.
https://www.unix.com/man-page/suse/1/qemu-img/ this is the man page of qemu-img in 2010
Then I run this at 15.3 as well and it passed all tests.

Reply:
Probably NO. SLES 15.3 is using `qemu-img` version 5.2.0, in which the usage is already valid.

```
rebase [--object objectdef] [--image-opts] [-U] [-q] [-f fmt] [-t cache] [-T src_cache] [-p] [-u] -b backing_file [-F backing_fmt] filename
```